src/services/item/plugins/app/appAction/schemas.ts

```diff
@@ -1,4 +1,6 @@
+import { customType } from '../../../../../plugins/typebox';
 import { accountSchemaRef } from '../../../../account/schemas';
+import { itemIdSchemaRef } from '../../itemLike/schemas';
```

Comment:
Should this come from `/item/schemas` instead?

Reply:
I'll move it on the refactor of Items Schemas 👍🏻